modules/lz4/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_LZ4)
+
+  set(LZ4_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${LZ4_DIR}/extra)
+  find_package(lz4 REQUIRED)
```

Comment:
Not sure the intention of those lines ?
From what I can see, lz4 repo doesn't provide an lz4 CMake package.

```suggestion
```